data/en/ships.json

```diff
@@ -292,70 +292,65 @@
 	"Saratoga":"Saratoga",
 	"Commandant Teste":"Commandant Teste",
 
-	"防空棲姫":"Air Defense Princess",
-	"空母棲鬼":"Aircraft Carrier Demon",
-	"空母棲姫":"Aircraft Carrier Princess",
-	"空母水鬼":"Aircraft Carrier Water Demon",
+	"イ":"I",
+	"ロ":"Ro",
+	"ハ":"Ha",
+	"ニ":"Ni",
+	"ホ":"Ho",
+	"ヘ":"He",
+	"ト":"To",
+	"チ":"Chi",
+	"リ":"Ri",
+	"ル":"Ru",
+	"空母ヲ":"Standard Carrier Wo",
+	"ヌ":"Nu",
+	"ワ":"Wa",
+	"カ":"Ka",
+	"ヨ":"Yo",
+	"タ":"Ta",
+	"レ":"Re",
+	"ソ":"So",
+	"ツ":"Tsu",
+	"ネ":"Ne",
+
+	"ナ":"Na",
+	"ラ":"Ra",
+	"ム":"Mu",
+	"ウ":"U",
+	"ヰ":"Wi",
+
+	"空母":"Aircraft Carrier",
+	"戦艦":"Battleship",
+	"駆逐":"Destroyer",
+	"重巡":"Heavy Cruiser",
+	"軽巡":"Light Cruiser",
+	"潜水":"Submarine",
+	"水母":"Seaplane Tender",
+
 	"飛行場姫":"Airfield Princess",
-	"泊地棲鬼":"Anchorage Demon",
-	"泊地棲姫":"Anchorage Princess",
-	"泊地水鬼":"Anchorage Water Demon",
 	"装甲空母鬼":"Armored Carrier Demon",
 	"装甲空母姫":"Armored Carrier Princess",
```

Comment:
Demon and Princess should be in suffix so 331-332 can be merged into "Armored Carrier"